### PR TITLE
Bumps up bundler version on travis

### PR DESCRIPTION
**Pull request by discorick**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,12 +1,12 @@
 
 env:
-  - GH_USER=gheeci GH_REPO=ghee_test GH_ORG=ghee-org TOKEN=a4cbd4976070d1cac1cbdc01de3e105116a70e83
```
Comment (dahlbyk) [inline .travis.yml]: Has this token been revoked?

Comment (discorick) [inline .travis.yml]: Yes sir! Actually more factually it was expired.

**Comment by dahlbyk** [comment]
Is this good to go? Should I not expect the build to be green?

**Comment by discorick** [comment]
Yeah should be good to go! No, not expecting a green build yet, will follow up later with a PR that ensures all the current specs are passing (likely just a configuration issue)

**Pull request merged**